code/modules/surgery/organs/augments_arms.dm

```diff
@@ -334,3 +334,90 @@
 		/obj/item/surgical_drapes,
 		/obj/item/knife/combat/cyborg,
 	)
+
+/obj/item/organ/internal/cyberimp/arm/muscle
+	name = "empovered musculature implant"
+	desc = "A cybernetic implant that empowers the strength of a human arm."
+	icon_state = "muscle_implant"
+
+	zone = BODY_ZONE_R_ARM
+	slot = ORGAN_SLOT_RIGHT_ARM_AUG
+
+	actions_types = list()
+
+	var/punch_damage = 13 ///The amount of damage dealt by your punches. Not really high, but better then normall punches.
+	var/gentle = TRUE //Will your throws be gentle or no
+	var/attack_throw_range = 1 ///How far away your attack will throw your oponent
+	var/throw_power_min = 1
+	var/throw_power_max = 4
+
+	var/emp_base_duration = 9 SECONDS ///How long will the implant malfunction if it is EMP'd
+
+/obj/item/organ/internal/cyberimp/arm/muscle/Insert(mob/living/carbon/reciever, special = FALSE, drop_if_replaced = TRUE)
+	. = ..()
+	if(ishuman(reciever)) //Sorry, only humans
+		RegisterSignal(reciever, COMSIG_HUMAN_EARLY_UNARMED_ATTACK, PROC_REF(on_attack_hand))
+
+/obj/item/organ/internal/cyberimp/arm/muscle/Remove(mob/living/carbon/implant_owner, special = 0)
+	. = ..()
+	UnregisterSignal(implant_owner, COMSIG_HUMAN_EARLY_UNARMED_ATTACK)
+
+/obj/item/organ/internal/cyberimp/arm/muscle/emp_act(severity)
```

Comment:
should have a balloon_alert in here, something like:

```txt
balloon_alert(user, "your arm spasms wildly!")
```

just to be clear to the viewer that it is indeed affected

Reply:
Added it